interface/views.py

```diff
@@ -71,7 +71,21 @@ def upload(request, course_code, assignment_code):
         form = UploadFileForm(request.POST, request.FILES)
         if form.is_valid():
             file = request.FILES['file']
-            handle_submission(file, assignment, request.user)
+
+            delta_t = 0
+            try:
+                handle_submission(file, assignment, request.user)
+            except TooManySubmissionsError as e:
+                delta_t = e.get_delta_t()
+                return render(
+                    request,
+                    'interface/upload.html',
+                    {
```

Comment:
Q: Does this looks ok? (as indentations?)

Reply:
Yeah that works